specification/redis/resource-manager/Microsoft.Cache/preview/2019-07-01/redis.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "RedisManagementClient",
     "description": "REST API for Azure Redis Cache Service.",
-    "version": "2019-07-01-preview"
+    "version": "2019-07-01"
```

Comment:
Is 2019-07-01 still a preview version? If not, I think you should not put them into the preview folder.